test/torch/tensors/test_replicated_shared.py

```diff
@@ -265,3 +265,19 @@ def test_sub(workers):
 
     z = (c - x).reconstruct().float_prec()
     assert torch.allclose(z, (c - t))
+
+
+def test_remote_share(workers):
+    bob, alice, james = (workers["bob"], workers["alice"], workers["james"])
+    x = torch.tensor([1, 2, 3], dtype=torch.long)
+
+    ptr = x.send(bob)
+    rst_pointer = ptr.share(bob, alice, james, protocol="falcon")
+
+    rst_pointer = rst_pointer + 10
+    rst = rst_pointer.get().child
+
+    # Validate all pointers are pointing to valid objects
+    rst.retrieve_shares()
+
```

Comment:
Can you also test call .share before .send if it works? (otherwise just leave it as a TODO)

Reply:
Hmm...isn't it tested at line 275? (I send a pointer and call share on it?)

Reply:
I was saying call .share() and then call .send()